src/auth-service/utils/mailer.js

```diff
@@ -170,6 +183,12 @@ const mailer = {
     next
   ) => {
     try {
+      const checkResult = await SubscriptionModel(
+        tenant
+      ).checkNotificationStatus(email, "email");
+      if (!checkResult.success) {
```

Comment:
@BenjaminSsempala just introducing a check in each of our mailer functions to only allow emails to go out to users who did not unsubscribe from our emails.